tmk_core/common/keyboard.h

```diff
@@ -67,6 +67,8 @@ void keyboard_init(void);
 void keyboard_task(void);
 /* it runs when host LED status is updated */
 void keyboard_set_leds(uint8_t leds);
+/* it runs whenever code has to behave differently on a slave */
+bool is_keyboard_master(void);
```

Comment:
Should probably be ifdef'd.

```suggestion
#ifdef SPLIT_KEYBOARD
bool is_keyboard_master(void);
#endif
```

Reply:
Definitely not. If it's not a split version then is_keyboard_master returns true unconditionally (there's a default weak version in keyboard.c that came in with ergodox infinity) and everything works as required.